jest.config.js

```diff
@@ -25,14 +26,14 @@ module.exports = {
     'endUserMessages\\.js',
     'pipelineStatusValues\\.js',
   ],
+  testEnvironment: 'jsdom',
   testPathIgnorePatterns: [
     '.*\\.mock\\.js',
     'test-utils',
   ],
   transform: {
     '^.+\\.(js|jsx|ts|tsx)$': '<rootDir>/node_modules/babel-jest',
     '^.+\\.css$': '<rootDir>/config/jest/cssTransform.js',
-    '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)': '<rootDir>/config/jest/fileTransform.js',
```

Comment:
So jest doesnt need this transform function anymore? Was this ran before for every file that is not js|jsx|ts|tsx|css|json ?

Reply:
Seems like we don't need it anymore. I will also delete `fileTransform.js` because it doesn't seem to be referenced anywhere else